package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "ms-rest-js",
+  "name": "@azure/ms-rest-js",
   "author": {
     "name": "Microsoft Corporation",
     "email": "azsdkteam@microsoft.com",
     "url": "https://github.com/Azure/ms-rest-js"
   },
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
There's no need to bump the version number because we're publishing to a new package, right?

Reply:
@daschult IMHO: It's no required but it's good to have higher version if we rename the package.